sdk/transaction.go

```diff
@@ -85,11 +122,11 @@ func (txs *TransactionService) AnnounceAggregateBondedCosignature(ctx context.Co
 	return txs.announceTransaction(ctx, &dto, announceAggregateCosignatureRoute)
 }
 
-// returns TransactionStatus for passed transaction id or hash
+// GetTransactionStatus returns TransactionStatus for passed transaction id or hash
 func (txs *TransactionService) GetTransactionStatus(ctx context.Context, id string) (*TransactionStatus, error) {
```

Comment:
Change the type of field "Group" in TransactionStatus from string to TransactionGroup please=) And use group here